### ISSUE TITLE
Unable to run Ubuntu on WSL or Bash as an administrator

### ISSUE THREAD

**Issue by harjeet-m** [microsoft/WSL#5727]
Hi Team,

I've enabled WSL on my Windows 10 machine running on OS build 19041.388. I have installed Ubuntu on top of it. While I'm trying to download SSL Root Certs, it does not allow shows error (screenshot attached). I've figured that this is because it requires elevated access to download and save the certs in .root directory. So while I try to run Ubuntu on WSL as an administrator, it throws an error (screenshot attached). Also, it does not let me run Bash as an administrator, the Bash windows just flashes and disappears.

I need a fix to run either of these as an administrator to download SSL certs.

Thanks,
Harjeet
![SSL CLI Cert Error](https://user-images.githubusercontent.com/40068189/89527098-68549b00-d806-11ea-8653-f235580ec098.png)
![Ubuntu Error](https://user-images.githubusercontent.com/40068189/89527131-786c7a80-d806-11ea-8688-b0731cc06a7f.jpg)

**Comment by therealkenc** [comment]
Please provide the CLI steps leading up to `sudo ssltool-cli`, and also the output of `cat /prov/version`.

> So while I try to run Ubuntu on WSL as an administrator

Do not run WSL as Windows Admin.

**Comment by therealkenc** [comment]
![image](https://user-images.githubusercontent.com/7049700/89566752-7b9b4280-d7d5-11ea-8602-b8676975f27d.png)

**Comment by harjeet-m** [comment]
> Please provide the CLI steps leading up to `sudo ssltool-cli`, and also the output of `cat /proc/version`.
> 
> > So while I try to run Ubuntu on WSL as an administrator
> 
> Do not run WSL as Windows Admin.

I'm running Ubuntu or Bash as an administrator as the SSL Root Cert it's trying to download needs access to the ./root directory of Ubuntu distro running on WSL.

The command ending to` sudo ssltool-cli` is following:
`sudo curl -o /usr/local/bin/ssltool-cli https://ssltool.rackspace.com/cli/ssltool-cli && sudo chmod +x /usr/local/bin/ssltool-cli`

This command is specific to the internal environment for the organization I'm working with so will not work in your test environment.

What I've found is the error is appearing because the Bash is not running as an administrator and does not have modify or write access to ./root directory. 

So currently I need to know how can I run Bash as an administrator so that it allows Certs to download in the ./root directory.

**Comment by therealkenc** [comment]
Windows Administrator is not Linux root.

I can't resolve `ssltool.rackspace.com`, as you mention, so I can't tell you the cause of the error on the screencap.

**Comment by therealkenc** [comment]
Did you install your WSL distribution from the Store, or was the distribution installed [manually](https://docs.microsoft.com/en-us/windows/wsl/install-manual)?

Are you able to start WSL with `wsl.exe` (or `wsl.exe -d Ubuntu-20.04`) when run from an elevated `cmd.exe` prompt, logged into Windows as user 'you'? Looks like:

![image](https://user-images.githubusercontent.com/7049700/89666741-c5486380-d88f-11ea-9924-61a71f2b2d5c.png)

If not, please collect [networking logs](https://github.com/microsoft/WSL/blob/master/CONTRIBUTING.md#9-networking-logs) and submit via the feedback hub.

Another approach would be do the `ssltool-cli` operation in `/mnc/c/Users/you/somewhere`, and then copy the result over to `C:\WINDOWS\system32` on the Windows side.

**Comment by eliasmarcelin0** [comment]
> Did you install your WSL distribution from the Store, or was the distribution installed [manually](https://docs.microsoft.com/en-us/windows/wsl/install-manual)?
> 
> Are you able to start WSL with `wsl.exe` (or `wsl.exe -d Ubuntu-20.04`) when run from an elevated `cmd.exe` prompt, logged into Windows as user 'you'? Looks like:
> 
> ![image](https://user-images.githubusercontent.com/7049700/89666741-c5486380-d88f-11ea-9924-61a71f2b2d5c.png)
> 
> If not, please collect [networking logs](https://github.com/microsoft/WSL/blob/master/CONTRIBUTING.md#9-networking-logs) and submit via the feedback hub.
> 
> Another approach would be do the `ssltool-cli` operation in `/mnc/c/Users/you/somewhere`, and then copy the result over to `C:\WINDOWS\system32` on the Windows side.

TLDR: WSL/Ubuntu only runs as admin. Doesn't matter what terminal or program I open it, that terminal or program must be ran as Admin or I get the errors below:

Errors: (I LISTED THESE even if they don't matter. These are the errors I get if I DON'T run WSL in an elevated state)
WSL.exe:
Doesn't even open unless as admin.

Ubuntu:
Installing, this may take a few minutes...
WslRegisterDistribution failed with error: 0x80070490
Error: 0x80070490 Element not found.

Vscode:
Element not found. 
Error code: Wsl/ERROR_NOT_FOUND


I just now have this problem and I'm not sure how much I've already changed looking at fixes online. But I've just realized that I can't open WSL/Unbuntu without any admin rights. This happens every way to open it. Opening from cmd/Powershell requires running as admin. Opening wsl.exe or ubuntu.exe requires being run as admin. Even VScode requires it. 

At least my Vscode is working again with Admin rights. At first, it wasn't even working, even as admin, but something changed as I was looking for solutions online.
